src/aura/HH_Container/HH_ContainerHelper.js

```diff
@@ -1,5 +1,14 @@
 ({
     /*******************************************************************************************************
+     * @description called at onInit to detect if Locker Service is enabled, warning if so.
+     */
+    warnOnLockerService: function(component) {
+        if (component.$lskey != null) {        
+            this.displayUIMessage(component, "LockerService is activated which prevents this page from working correctly.  Under Setup, search for Critical Updates, and deactivate the critical update 'Enable Lightning LockerService Security'. ", "divUIMessageContainer");
```

Comment:
I know that this is temporary and hopefully very short lived, so maybe it doesn't make sense to put it in a label to localize it.  But, should we get doc review on this string?